### PR TITLE
build-outlook-clone-with-react-hooks

### PR DESCRIPTION
**Pull request by lalith1403**
With respect to the earlier PR, I have incorporated all the changes mentioned. The PR is a duplicate one to avoid merge conflicts.

Earlier Description:
I have checked the above file with respect to code, and grammar on grammarly.com and hemingwayapp.com. There is just one picture to be included, which is hero.jpg, included in the repo. I completely discuss react hooks and have written optimized code and explained it all.

Author name: Lalithnarayan C
Description: Building outlook clone using react hooks. Optimizing the codebase by minimzing the number of computations, number of re-renders and using styled-components.

### CODE REVIEW

File: articles/build-an-outlook-clone-using-react-hooks/index.md [hunk 1]
```diff
@@ -0,0 +1,895 @@
...
+author: lalithnarayan-c
+date: 2020-08-09T00:00:00-10:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/build-an-outlook-clone-using-react-hooks/hero.jpg
+    alt: output image outlook clone example
+---
+Welcome to part two of the React series. In this tutorial, we will cover the same outlook clone that we built in [part one](/engineering-education/build-an-outlook-clone-using-react), but this time we will use React Hooks. The earlier codebase was repetitive and there is a scope for optimization of the entire project. To skip the theory and get straight into coding, [click here](#Code)
```
Comment (hectorkambow) [inline articles/build-an-outlook-clone-using-react-hooks/index.md:17]: the click here tag does not redirect - not sure if I messed anything up - please check and revise.  👍 @lalith1403

Comment (lalith1403) [reply articles/build-an-outlook-clone-using-react-hooks/index.md:17]: It is working! Both on my PC as well as on GitHub. 

Check this link: https://github.com/section-io/engineering-education/blob/abeb74e146e22158033f152d1107559616c6dadb/articles/build-an-outlook-clone-using-react-hooks/index.md

The link works...

**Comment by hectorkambow** [comment]
Duplicate of this [PR](https://github.com/section-io/engineering-education/pull/174#issuecomment-671464492)

**Comment by hectorkambow** [comment]
@ninjaginja This article should be ready to merge - it was given a tech review by Marcus on the duplicate PR - Only thing if you could check the second link (here link) in the article - it doesn't seem to jump to the proper place in the article for me - it seems to be working for the student. 
🤔

**Comment by ninjaginja** [comment]
@hectorkambow Those links are case-sensitive in Hugo :)

**Pull request merged**